Makefile

```diff
@@ -3,7 +3,7 @@ BIN_NAME=terraform-provider-instaclustr
 
 
 # for VERSION, don't add prefix "v", e.g., use "1.9.8" instead of "v1.9.8" as it could break circleCI stuff
-VERSION=1.14.3
+VERSION=1.14.5
```

Comment:
Why is this being bumped up by 2?

Reply:
I saw there is another PR marked as 1.14.4. I think it might be go into preprod before me.

Reply:
Ah I see. You'll want to sync with releases then since you won't be able to merge your branch if it's not up to date